test/schemas/sdkSchemas.test.ts

```diff
@@ -64,6 +64,141 @@ describe('The event metadata schema', () => {
     });
 });
 
+describe('The cookie options schema', () => {
```

Comment:
As a housekeeping, I think we can refactor these data providers to generate more semantic copies